implants/lib/eldritch/src/runtime/eval.rs

```diff
@@ -231,7 +231,21 @@ impl Runtime {
         let module: Module = Module::new();
         let mut input_params: Dict = Dict::new(SmallMap::new());
 
+        let mut files_type = "remote_assets";
         for (key, value) in &tome.parameters {
```

Comment:
is tome parameters the k/v's defined in the metadata.yaml file? If so it seems like this is exposing configuration of remote, local, embedded assets there, but i'd expect it to be a command line argument.

Reply:
I currently have it strip out all parameters with the `__` prefix. This prevents info from being in the params dict at runtime. I do agree its a hacky way, but currently lib/eldritch does not expose any other way to pass things into the runtime. Other solutions could be

1. Adding a globals param to `alloc_module` - This would break existing calls and touch more code
2. Adding a field to the Tome protobuf - Seems like a big change, but it might be useful down the road to inject globals into a tome when its run
3. Somehow getting the CLI args from inside the `alloc_module` function, But this seems like it should be done
4. Use existing Tome fields to pass the data in